src/Ganeti/BasicTypes.hs

```diff
@@ -206,12 +206,12 @@ instance MonadTrans (ResultT a) where
 instance (MonadIO m, Error a) => MonadIO (ResultT a m) where
   liftIO = ResultT . liftIO
                    . liftM (either (failError . show) return)
-                   . (try :: IO a -> IO (Either IOError a))
+                   . (try :: IO α -> IO (Either IOError α))
```

Comment:
Sorry for being late.

What has changed here? Latin letter `a` to Greek letter `α` (alpha)? Is this intentional? And if so, why?

Reply:
This is presumably to decouple the inner (`IO a`) from the outer (`Error a`) type, letting type inference do its magic. @mbakke Greek letters are great (obviously, I'm Greek :-) but it's rather unusual to encounter them in source code. I would stick with ASCII characters if possible, e.g. we could try using `a'` here.